examples/nlp/question_answering/question_answering_squad.py

```diff
@@ -246,6 +247,51 @@ def create_pipeline(
     )
 
 
+MODEL_CLASSES = {
```

Comment:
this looks very general and could be used in other examples. to avoid copying it multiple times, should we move it to utils?